languages/scala/exercises/concept/operators/Rectangle.scala

```diff
@@ -0,0 +1,5 @@
+case class Rectangle(width: Int, height: Int) {
+  def +(that: Rectangle): Rectangle = ???
+
+  def *(factor: Int): Rectangle = ???
```

Comment:
I'm slightly torn between having stubs here (like you have now) or to put TODO comments here. In general we want there to be stubs, so that great. We usually only have TODOs when the exercise introduces new syntax and we want to make sure the student writes the syntax from scratch.

It looks like the operator overloading doesn't really introduce new syntax, but by having the stubs here I feel like we might be giving away too much of the solution. I'm slightly leaning towards having TODOs here (which is also what we do in the [C# `operator-overloading` exercise](https://github.com/exercism/v3/blob/master/languages/csharp/exercises/concept/operator-overloading/OperatorOverloading.cs#L12).

What do you think?

Reply:
Makes definitely sense, I had already same thoughts, that adding these function definitions gives already half of the solution. 
I actually wanted the unit tests to compile in order to support the student with solving the tasks one by one.
But in this case it makes sense to go with TODOs instead. Will adapt that.

Reply:
👍